dcos/mesos.py

```diff
@@ -40,8 +40,21 @@ def __init__(self):
         else:
             self._mesos_master_url = mesos_master_url
 
+    def get_dcos_url(self, path):
```

Comment:
why do we allow a user to instantiate without dcos_url? What can they do without it?